concent_api/core/tests/test_integration_subtask_results_verify.py

```diff
@@ -216,7 +216,7 @@ def test_that_concent_reponds_with_too_small_requestor_deposit_when_requestor_do
                 )
 
         is_account_status_positive_mock.assert_called_with(
-            client_eth_address=self.report_computed_task.task_to_compute.requestor_ethereum_address
+            client_eth_address=self.report_computed_task.task_to_compute.requestor_ethereum_address  # pylint: disable=no-member
```

Comment:
Removing this and other disables in this file doesn't produce any warnings, so please remove them.

Reply:
```
[PYLINT: concent_api]
Using config file /home/jakub/Documents/work/projects/concent/pylintrc
************* Module core.tests.test_integration_subtask_results_verify
E:219,31: Instance of 'SubFactory' has no 'requestor_ethereum_address' member (no-member)
E:321,31: Instance of 'SubFactory' has no 'requestor_ethereum_address' member (no-member)
E:327,26: Instance of 'SubFactory' has no 'compute_task_def' member (no-member)
E:328,23: Instance of 'SubFactory' has no 'compute_task_def' member (no-member)
```

Reply:
Can it be different pylint versions, or something like that?
```
pylint==1.8.3
pylint-django==0.9.4
pylint-plugin-utils==0.2.6
```

Reply:
I had 1.8.2 and it was not showing this, but after i have run `pip install --upgrade pylint` to 1.9.1 the warning was there. So keep it as it is, but I think we should update the minimum version of pylint to the one that is not creating ambigious warnings. So please update to lates 1.9.1, it gives no warnings for current code.

Reply:
I've updated pylint to 1.8.4 and pylint-django to 0.11 and I don't see those warnings anymore. We need to do something to our dev requirements to avoid that bs.

Reply:
Yes exactly, update to the latest versions, fix code and update minimum versions in requirements-dev.

Reply:
Done